src/components/Daos/Daos.scss

```diff
@@ -19,29 +20,31 @@
     font-weight: bold;
     border-bottom: 1px solid rgba(195, 212, 231, 1);
     padding: 0;
-    margin-bottom: 0;
+    margin: 0;
   }
 }
 
 .searchBox {
-  position: relative;
-  margin-top: 15px;
+  position: sticky;
+  display: inline-block;
+  top: 65px;
+  z-index: 1000000000;
 
   span {
     color: #4f6176;
   }
 
   input {
-    font-size: 22px;
+    font-size: 15px;
     width: 425px;
     padding: 2px;
   }
 }
 
 .createDaoButton {
-  position: fixed;
-  right: 20px;
-  top: 122px;
+  position: sticky;
+  float: right;
```

Comment:
We can pretty much always do things better without using float.  Dunno what you're attempting here, but can it not be done with css flex styles?

Reply:
maybe it could. but this works